### PR TITLE
Add natspec for identity contracts

### PR DESCRIPTION
**Pull request by berndbohmeier**

### CODE REVIEW

**Comment by cducrest** [comment]
https://github.com/trustlines-protocol/contracts/pull/307/files#diff-1d2406c6378a088398d220dd3c7faa27R28
Maybe use three slashes `///` instead of two.

File: contracts/identity/Identity.sol [hunk 1]
```diff
@@ -39,12 +48,41 @@ contract Identity is ProxyStorage {
...
+              `feeRecipient` if set, or `msg.sender`. Only has to be payed, if the meta-transaction succeeds.
+     * @param gasPrice Price per 1M gas in the selected currency network to be payed by this contract to
+              `feeRecipient` if set, or `msg.sender`. Only has to be payed, if the meta-transaction succeeds.
+     * @param gasLimit Maximum amount of gas the meta-transaction can consume and the sender is willing to pay.
+     * @param feeRecipient Recipient of the meta-transaction fee, or the zero address. If set to the zero address
+              the fee goes to msg.sender.
+     * @param currencyNetworkOfFees CurrencyNetwork in which to pay the fees.
+     * @param nonce Number used once for replay protection. Can either be an increasing number to enforce the ordering
+              of meta-transaction, and then must be equal to lastNonce + 1,
+              or can be a random number >= `maxNonce` if the order does not matter.
```
Comment (cducrest) [inline contracts/identity/Identity.sol]: Add that if `nonce >= maxNonce` we use the hash as a replay mechanism

File: contracts/identity/Identity.sol [hunk 2]
```diff
@@ -114,12 +152,28 @@ contract Identity is ProxyStorage {
...
         require(msg.sender == owner || msg.sender == address(this), "Can only be called by owner or via meta-tx");
-        hashUsed[hash] = true;
-        emit TransactionCancellation(hash);
+        hashUsed[txHash] = true;
+        emit TransactionCancellation(txHash);
     }
 
+    /**
+     * Executes the given function call `data` at `to` and sends `value` coins
+     * @dev Executes a function in the name of this identity. Reverts if the function call fails. Can be only called
```
Comment (cducrest) [inline contracts/identity/Identity.sol]: `Can only be called`

File: contracts/identity/Identity.sol [hunk 3]
```diff
@@ -114,12 +152,28 @@ contract Identity is ProxyStorage {
...
+               by the owner of this identity.
+     * @param to The contract where to execute the function and where to send `value` coins
```
Comment (cducrest) [inline contracts/identity/Identity.sol]: `to` does not need to be a contract, it's just an address

File: contracts/identity/Identity.sol (hunk 1 above)
Comment (berndbohmeier) [reply contracts/identity/Identity.sol]: done

**Pull request merged**